test/jdk/java/awt/event/ComponentEvent/ListItemEventsTest.java

```diff
@@ -0,0 +1,193 @@
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+
+import java.awt.EventQueue;
+import java.awt.FlowLayout;
+import java.awt.Frame;
+import java.awt.List;
+import java.awt.Point;
+import java.awt.Rectangle;
+import java.awt.Robot;
+import java.awt.event.FocusAdapter;
+import java.awt.event.FocusEvent;
+import java.awt.event.InputEvent;
+import java.awt.event.KeyEvent;
+import java.awt.event.MouseEvent;
+import java.util.concurrent.CountDownLatch;
+import java.util.concurrent.TimeUnit;
+
+/*
+ * @test
+ * @key headful
+ * @bug 8295774
+ * @summary Verify that List Item selection via mouse/keys generates ItemEvent/ActionEvent appropriately.
+ * @run main ListItemEventsTest
+ */
+public class ListItemEventsTest {
+
+    private static final int waitDelay = 1000;
+
+    private volatile static Frame frame;
+    private volatile static List list;
+    private volatile static boolean actionPerformed = false;
+    private volatile static boolean itemStateChanged = false;
+    private volatile static Robot robot;
+    private volatile static CountDownLatch latch;
+
+    public static void initializeGUI() {
+        frame = new Frame("Test Frame");
+        frame.setLayout(new FlowLayout());
+        list = new List();
+        list.add("One");
+        list.add("Two");
+        list.add("Three");
+        list.add("Four");
+        list.add("Five");
+        list.addItemListener((event) -> {
+            System.out.println("Got an ItemEvent:" + event);
+            itemStateChanged = true;
+        });
+        list.addActionListener((event) -> {
+            System.out.println("Got an ActionEvent:" + event);
+            actionPerformed = true;
+        });
+        list.addFocusListener(new FocusAdapter() {
+            public void focusGained(FocusEvent event) {
+                System.out.println("Got an FocusEvent:" + event);
+                latch.countDown();
+            }
+        });
+
+        frame.add(list);
+        frame.pack();
+        frame.setLocationRelativeTo(null);
+        frame.setVisible(true);
+    }
+
+    public static void main(String[] s) throws Exception {
+        latch = new CountDownLatch(1);
+        robot = new Robot();
+        try {
+
+            robot.setAutoDelay(100);
+            robot.setAutoWaitForIdle(true);
+
+            EventQueue.invokeLater(ListItemEventsTest::initializeGUI);
+            robot.waitForIdle();
+
+            Point listAt = list.getLocationOnScreen();
+            // get bounds of button
+            Rectangle bounds = list.getBounds();
```

Comment:
What button? Better remove the comment.

Since you need only the size, use [`getSize()`](https://docs.oracle.com/en/java/javase/17/docs/api/java.desktop/java/awt/Component.html#getSize())

Reply:
Resolved now code modified as below 
 
Dimension listDimension = list.getSize();